README.md

```diff
@@ -1 +1,1175 @@
+```
+
+ - Now you can build a Ballerina executable archive (.balx) of the service that we developed above, using the following command. It points to the service file that we developed above and it will create an executable binary out of that. 
+This will also create the corresponding docker image using the docker annotations that you have configured above. Navigate to the `message-transformation---ballerina/guide/` folder and run the following command.
+
+```
+   $ballerina build message_transformation
+   
+Compiling source
+    message_transformation.bal
+
+Generating executable
+    ./target/message_transformation.balx
+        @docker                  - complete 3/3 
+
+        Run the following command to start docker container:
+        docker run -d -p 9090:9090 ballerina.guides.io/message_transformation:v1.0
+```
+
+- Once you successfully build the docker image, you can run it with the `` docker run`` command that is shown in the previous step.  
+
+```   
+   $docker run -d -p 9090:9090 ballerina.guides.io/message_transformation:v1.0
+```
+
+- Here we run the docker image with flag`` -p <host_port>:<container_port>`` so that we use the host port 9090 and the container port 9090. Therefore you can access the service through the host port. 
+
+- Verify docker container is running with the use of `` $ docker ps``. The status of the docker container should be shown as 'Up'. 
+
+- You can access the service using the same curl commands that we've used above. 
+ 
+```
+curl -v http://localhost:9090/contentfilter -d '{"id" : 105, "name" : "ballerinauser", "city" : "Colombo 03", "gender" :    "male"}' -H "Content-Type:application/json" -X POST
+```
+
+### Deploying on Kubernetes
+
+- You can run the service that we developed above, on Kubernetes. The Ballerina language offers native support for running a Ballerina program on Kubernetes, with the use of Kubernetes annotations that you can include as part of your service code. Also, it will take care of the creation of the docker images. So you don't need to explicitly create docker images prior to deploying it on Kubernetes. Refer to [Ballerina_Kubernetes_Extension](https://github.com/ballerinax/kubernetes) for more details and samples on Kubernetes deployment with Ballerina. You can also find details on using Minikube to deploy Ballerina programs. 
+
+Since this guide requires MySQL as a prerequisite, you need a couple of more steps to create a MySQL pod and use it with our sample.  
+
+First, let's look at how we can create a MySQL pod in kubernetes. If you are working with minikube, it will be convenient to use the minikube's in-built docker daemon and push the MySQL docker image we are about to build to the minikube's docker registry. This is because during the next steps, in the case of minikube, the docker image we build for employee_database_service will also be pushed to minikube's docker registry. Having both images in the same registry will reduce the configuration steps.
```

Comment:
Typo: kubernetes -> Kubernetes

Check in all the places

Reply:
Done